connect/routes/data.py

```diff
@@ -28,6 +28,7 @@ class LinuxForHealthDataRecordResponse(BaseModel):
 
     uuid: uuid.UUID
     lfh_id: str
+    operation: str
```

Comment:
we probably want operation to by any string, not just limited to the HTTP operations